spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaMessageChannelBinder.java

```diff
@@ -664,15 +539,15 @@ private KafkaMessageListenerContainer createMessageListenerContainer(KafkaProper
 		if (logger.isDebugEnabled()) {
 			logger.debug("Listening to topic " + topic);
 		}
-		// if we have less target partitions than target concurrency, adjust accordingly
-		messageListenerContainer.setConcurrency(Math.min(maxConcurrency, listenedPartitions.size()));
+		// if we have fewer target partitions than target concurrency, adjust accordingly
+		messageListenerContainer.setConcurrency(Math.min(consumerProperties.getConcurrency(), listenedPartitions.size()));
 		OffsetManager offsetManager = createOffsetManager(group, referencePoint);
-		if (resetOffsets) {
+		if (consumerProperties.isResetOffsets()) {
 			offsetManager.resetOffsets(listenedPartitions);
 		}
 		messageListenerContainer.setOffsetManager(offsetManager);
-		messageListenerContainer.setQueueSize(accessor.getProperty(QUEUE_SIZE, defaultQueueSize));
-		messageListenerContainer.setMaxFetch(accessor.getProperty(FETCH_SIZE, defaultFetchSize));
+		messageListenerContainer.setQueueSize(defaultQueueSize);
```

Comment:
I wonder if we can remove `default` from the variable names for all the `binder` configuration properties. Having `default` makes it look like constant values in a way. If we remove `default`, the `setter` and `getter` methods on the binder would look better (like setQueueSize, setFetchSize) on the binder.

This also means that `binder` configuration properties don't need to carry `.default` in them (as they are the actual property values and not the default ones)?